index.html

```diff
@@ -4,6 +4,14 @@
     <meta charset="UTF-8">
```

Comment:
**suggestion**: Consider moving the CSS to an external stylesheet for better separation of concerns and maintainability.

Reply:
i will after finishing the mvp. this is far from being important for now